pkg/kubectl/main.go

```diff
@@ -15,6 +16,9 @@ import (
 )
 
 func Main() {
+	if runtime.GOOS == "windows" {
+		os.Args = os.Args[1:]
```

Comment:
I don't think we need to this for Windows. You are doing it to support "k3s kubectl <command>", but it will not work anymore for the symbol-linked "kubectl.exe".

crictl has the same issue.

Reply:
sorry, invoking the symbol-linked kubectl may not a valid scenario. Please ignore me.